app/src/free/res/values/donottranslate.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">Shuttle</string>
+    <string name="app_name">MUSER</string>
```

Comment:
Note that this PR contains two commits - that usually means that you didn't check out the main branch (`usf`) again before creating a new branch for this particular feature (version bump).

Could you please re-submit PR with only changes for the version number?

One way to do this is:
* `git checkout usf`
* `git pull origin usf` // Updates your local branch with latest changes on GitHub branch
* `git checkout -b bump-version2` // Create new branch
* `git cherry-pick 6f652a572ffb695c6808890dc351217b1fe6da3e` // "cherry-pick" the commit that changed the version numbers into your current branch
* `git push origin -f bump-version` // force-push your current branch to the existing bump-version branch (this PR), so your current branch replaces the current contents of this PR.

There are other ways to accomplish this too via Git (e.g., rebasing this branch on `usf` after you pull the `usf` branch from GitHub, and then force-push).